installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -6,9 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
-
 ifndef::openshift-origin[]
+You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
```

Comment:
Since 4.15 also agent-based installer is available as an installation approach. Could be worth mentioning it here?

Reply:
@andfasano if that's possible I'd add that in a separate PR. We can open a GitHub ticket for that, so that it's publicly available and the community can pick up the task. Is it viable? What do you think @mburke5678?

Reply:
@peterroth Sounds good to me. How do we want to proceed with this PR?

Reply:
@mburke5678 I can't see blockers, so if there is no veto or open task, merge this one.
The new ticket should reference this one to give context for the necessity of documenting the agent based installation.